src/lib/mina_generators/parties_generators.ml

```diff
@@ -1269,11 +1253,10 @@ let gen_parties_from ?failure ?(max_other_parties = max_other_parties)
         let%bind party0 =
           (* Signature authorization to start *)
           let authorization = Control.Signature Signature.dummy in
-          let required_balance_change = Currency.Amount.Signed.zero in
-          gen_party_from ~account_ids_seen ~update ?failure ~authorization
-            ~new_account:new_parties ~permissions_auth ~zkapp_account
-            ~available_public_keys ~required_balance_change ~ledger
-            ~account_state_tbl ?protocol_state_view ?vk ()
+          gen_party_from ~zkapp_account_ids ~account_ids_seen ~update ?failure
+            ~authorization ~new_account:new_parties ~permissions_auth
```

Comment:
rename `new_parties` to something that says it is a boolean? maybe `is_new_party`?

Reply:
The `new_parties` variable would be replaced by another variable in my token accounts PR. So I think it's ok to keep it this way now to reduce merge conflict between my 2 PRs.